libs/extensions/angular/sidebar-menu/src/components/menu-item-list/menu-item-list.component.html

```diff
@@ -1,85 +1,14 @@
-    } @else if (item.children) {
-      @let submenuId = `item-${item.id}-content`;
-      <button
-        type="button"
-        class="menu-item menu-item-{{ level }} togglable-menu-item"
-        [attr.id]="'item' + item.id"
-        [attr.aria-controls]="submenuId"
-        [attr.aria-expanded]="item.isExpanded"
-        (click)="vm.toggleSubmenu(item)"
-      >
-        <kirby-x-menu-item-content
-          [title]="item.title ?? ''"
-          [icon]="item.icon"
-          [badge]="item.badge"
-          [isExpanded]="item.isExpanded ?? false"
-        ></kirby-x-menu-item-content>
-      </button>
-      @if (item.isExpanded) {
-        <div [@dropDownMenu] [@.disabled]="vm.disableAnimations()">
-          <kirby-x-menu-item-list
-            [items]="item.children"
-            [level]="level + 1"
-            [id]="submenuId"
-            [disableAnimations]="disableAnimations()"
-          ></kirby-x-menu-item-list>
-        </div>
-      }
-    } @else if (item.isAction) {
-      <a
-        class="menu-item menu-item-{{ level }} clickable-menu-item"
-        [class.selected]="item.selected ?? false"
-        [attr.id]="'item' + item.id"
-        [attr.href]="item.link?.url ?? ''"
-        (click)="$event.preventDefault(); vm.selectItem(item)"
-        (keydown.enter)="$event.preventDefault(); vm.selectItem(item)"
-      >
-        <kirby-x-menu-item-content
-          [title]="item.title ?? ''"
-          [icon]="item.icon"
-          [badge]="item.badge"
-        ></kirby-x-menu-item-content>
-      </a>
-    } @else if (item.link?.relativeLink) {
-      <a
-        class="menu-item menu-item-{{ level }} clickable-menu-item"
-        [class.selected]="item.selected ?? false"
-        [attr.id]="'item' + item.id"
-        [target]="item.link?.target ?? '_self'"
-        [routerLink]="item.link?.relativeLink"
-        [queryParams]="item.link?.queryParams"
-        [skipLocationChange]="item.skipLocationChange ?? false"
-        [replaceUrl]="item.replaceUrl ?? false"
-        (click)="vm.selectItem(item)"
-        (keydown.enter)="vm.selectItem(item)"
-      >
-        <kirby-x-menu-item-content
-          [title]="item.title ?? ''"
-          [icon]="item.icon"
-          [badge]="item.badge"
-        ></kirby-x-menu-item-content>
-      </a>
-    } @else if (item.link?.url) {
-      <a
-        class="menu-item menu-item-{{ level }} clickable-menu-item"
-        [class.selected]="item.selected ?? false"
-        [attr.id]="'item' + item.id"
-        [attr.href]="item.link?.url"
-        [attr.target]="item.link?.target"
-        (click)="vm.selectItem(item)"
-        (keydown.enter)="vm.selectItem(item)"
-      >
-        <kirby-x-menu-item-content
-          [title]="item.title ?? ''"
-          [icon]="item.icon"
-          [badge]="item.badge"
-        ></kirby-x-menu-item-content>
-      </a>
-    }
+@for (item of vm.items(); track item.id) {
+  @if (item.isDivider) {
+    <kirby-divider></kirby-divider>
```

Comment:
When in a list, divider needs to be hidden from assistive tech: https://665d7d16d6f1242580f1e9bf-oxhqhhyeig.chromatic.com/?path=/story/components-sidebar--default&addonPanel=storybook/a11y/panel&a11ySelection=violations.list.1 

This was previously achieved with `aria-hidden` before menu refactor, so probably still a decent solution here.
```suggestion
    <kirby-divider aria-hidden="true"></kirby-divider>
```

Reply:
done